app/presenters/digest_presenter.rb

```diff
@@ -0,0 +1,75 @@
+class DigestPresenter < NotificationPresenter
+  include AvatarHelper
+
+  attr_accessor :notifications, :template
+
+  def initialize(notifications, template)
+    @notifications = notifications
+    @template = template
+  end
+
+  def comment_path(anchor: false)
+    polymorphic_url(
+      path_to_comment,
+      anchor: anchor
+    )
+  end
+
+  def created_at_ago
+    # We can't use the local_time gem here because there's no JS
+    "#{time_ago_in_words(notification.created_at)} ago"
+  end
+
+  private
+
+  def avatar_image(size)
```

Comment:
here, we are overwriting the method we just included with `AvatarHelper` yes?

Reply:
yes, we can't use the original method because we 1) need to handle a non-existent user and 2) add a fallback image.

Reply:
can that be done in the original method and only have 1 `avatar_image` method ?

Reply:
I tried it out and it seems possible. However, there are gonna be some substantial changes that I think it belongs to another PR. For starters, the `avatar_image` in the helper wraps the image around a `span` tag while this presenter doesn't. The helper also forces a `style` attribute which we don't need in this presenter. Those and the fact that this refactor benefits the other presenters too makes me think it should be done in a different PR.

Reply:
I can get onboard with doing this as a different PR. It would be good to get all the presenters/views using the same helper, but it doesn't have to be done here.